final_task/pycalc/addmultsigns.py

```diff
@@ -0,0 +1,77 @@
+"""This module contains a class that allows to take into account implicit multiplication signs"""
+
+# import from pycalc self library
+from .pycalclib import constants, functions, negative_functions
+
+
+class Multsignsadder():
```

Comment:
Brackets are not necessary if there are no base classes.
You can write `class MultSignsAdder: `

Reply:
fixed